velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -292,12 +301,24 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
     {
       SelectivityVector selection(dictionarySize);
       DecodedVector decoded(*dictionaryVector, selection, false);
-      check(decoded);
+
+      // FIXME: To remove the if condition once a bug of wrapping the
```

Comment:
Let's apply to fix from https://github.com/facebookincubator/velox/issues/3704#issuecomment-1381582791 

and simplify this test to verify that the result of `wrap` is equal to the original vector:

```
    {
      SelectivityVector selection(dictionarySize);
      DecodedVector decoded(*dictionaryVector, selection, false);
      check(decoded);

      auto wrappedVector =
          decoded.wrap(base, *dictionaryVector, selection.end());
      assertEqualVectors(dictionaryVector, wrappedVector);
    }
```